cluster/addons/addon-manager/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## Version 9.0.3 (Wed May 13 2020 Antoni Zawodny <zawodny@google.com>)
+ - Enable overriding the default list of whitelisted resources
```

Comment:
This is not a bug fix. So it shouldn't be patch release imho.

I suggest 9.1.0 instead

Reply:
Done. Thanks for pointing that out.